test/unit/specs/components/ActionModal/components/__snapshots__/ActionModal.spec.js.snap

```diff
@@ -98,8 +98,7 @@ exports[`ActionModal should show the action modal when user has logged in with e
      
     <tableinvoice-stub
       amount="0"
-      gasestimate="0"
-      gasprice="2.5e-8"
+      estimatedfee="0"
```

Comment:
should this be 0?

Reply:
when the estimate was 0 before then yes ;)